src/powerhouse/components/tree-view-item/tree-view-item.tsx

```diff
@@ -80,7 +80,7 @@ export const TreeViewItem: React.FC<TreeViewItemProps> = props => {
                             name="caret"
                             className={twMerge(
                                 open && 'rotate-90',
-                                'ease delay-50 pointer-events-none transition',
```

Comment:
I was quite surprised by this catch from the tailwind eslint plugin. it's strange to me, but it turns out that the shortest delay that tailwind has is 75ms https://tailwindcss.com/docs/transition-delay